File: src_old/main/java/me/newyith/fortressOrig/util/Log.java
package me.newyith.fortressOrig.util;

import org.bukkit.Bukkit;
import org.bukkit.ChatColor;
import org.bukkit.command.ConsoleCommandSender;

public class Log {
	public static void success(String s) {
		sendConsole(s, ChatColor.GREEN);
	}

	public static void warn(String s) {
		sendConsole(s, ChatColor.YELLOW);
	}

	public static void error(String s) {
		sendConsole(s, ChatColor.RED);
	}

	public static void log(String s) {
		sendConsole(s, ChatColor.WHITE);
	}

	public static void sendConsole(String s, ChatColor color) {
		s = ChatColor.AQUA + "[FP] " + color + s;
		ConsoleCommandSender console = Bukkit.getServer().getConsoleSender();
		console.sendMessage(s);
	}
}

File: src/main/java/me/newyith/util/Log.kt
package me.newyith.util

import org.bukkit.Bukkit
import org.bukkit.ChatColor

class Log {
	companion object {
		fun success(s: String) {
			sendConsole(s, ChatColor.GREEN)
		}

		fun warn(s: String) {
			sendConsole(s, ChatColor.YELLOW)
		}

		fun error(s: String) {
			sendConsole(s, ChatColor.RED)
		}

		fun log(s: String) {
			sendConsole(s, ChatColor.WHITE)
		}

		fun sendConsole(s: String, color: ChatColor) {
			var s = s
			s = ChatColor.AQUA.toString() + "[FP] " + color + s
			val console = Bukkit.getServer().consoleSender
			console.sendMessage(s)
		}
	}
}